src/Etterna/Models/Misc/PlayerOptions.cpp

```diff
@@ -430,9 +431,8 @@ PlayerOptions::FromString(const std::string& sMultipleMods)
 	std::string sThrowAway;
 	for (auto& s : vs) {
 		if (!FromOneModString(s, sThrowAway)) {
-			LOG->Trace("Attempted to load a non-existing mod \'%s\' for the "
-					   "Player. Ignoring.",
-					   (s).c_str());
+			Locator::getLogger()->warn("Attempted to load a non-existing mod \'{}\' for the "
```

Comment:
This message occurs extremely often due to our hacky nature of handling the Rates mods. It essentially pops up once per rate on some Screens and also pops up once or more per Replay load in Evaluation. It may be better to make this a trace.

Reply:
Fixed.